var/spack/repos/builtin/packages/nalu-wind/package.py

```diff
@@ -42,7 +42,8 @@ class NaluWind(CMakePackage, CudaPackage):
     )
     variant("openfast", default=False, description="Compile with OpenFAST support")
     variant("tioga", default=False, description="Compile with Tioga support")
-    variant("hypre", default=False, description="Compile with Hypre support")
+    variant("hypre", default=True, description="Compile with Hypre support")
+    variant("trilinos-solvers", default=True, description="Compile with Trilinos Solvers support")
```

Comment:
According to the message in conflict at lines 67-69, it sounds like one or the other of these should be enabled but not both.  

If that is the case, then one of these should default to `False`.  

And a `conflicts` could be added to ensure both aren't enabled at the same time.

Reply:
The behavior we want is actually the opposite. At least one needs to be enabled at all time.

Reply:
Ah yes, that would be poor wording on my part. In fact it is ok to enable either of them, or both of them, but nalu-wind won't do anything useful if neither of them is enabled. I will incorporate Phil's suggestion below. Thanks!